### PR TITLE
Update olive.rst

### PR DESCRIPTION
**Pull request by cachob**
@sarina - thanks for your time teaching this!

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @cachob!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/openedx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.     If you've signed an agreement in the past, you may need to re-sign.  See     [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)     for details.

    
Once you've signed the CLA, please allow 1 business day for it to be processed.  After this time, you can re-run the     CLA check by editing the PR title.  If the problem persists, you can tag the `@openedx/cla-problems` team in a     comment on your PR for further assistance.

File: source/community/release_notes/olive.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ These are the release notes for the Olive release, the 15th community release of
  :depth: 1
  :local:
 
-Breaking Changes
+Breaking Bad Changes
 ================
```
Comment (sarina) [inline source/community/release_notes/olive.rst:17]: This line needs to be the same length as the words above

See https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html for how to make changes to your pull request!

**Comment by mphilbrick211** [comment]
Hi @cachob! Friendly reminder on the CLA form - were you able to submit? If you submitted it, the CLA check should turn green when you re-run the tests. Please let Sarina or I know if you have any questions. Thanks!

**Comment by sarina** [comment]
I think actually @cachob - I'm going to close this Pull Request. The reason is that "Breaking Changes" is a common term across our documentation. Changing this to "Breaking Bad Changes" may cause more confusion. Plus, often breaking changes actually mean something good - that we've added cool new functionality.

**Comment by openedx-webhooks** [comment]
@cachob Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/docs.openedx.org&entry.867055334=https://github.com/openedx/docs.openedx.org/pull/292&entry.1484655318=https://github.com/cachob&entry.752974735=2023-03-28+18:40&entry.1917517419=2023-04-04+16:43&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**